manifests/charts/base/crds/maistra.io_meshfederations.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.2.4
+    controller-gen.kubebuilder.io/version: (devel)
```

Comment:
@jwendell, any idea what this is about?

Reply:
@rcernich this is likely because we run controller-gen using `go run vendor/...` instead of running a proper compiled binary.
https://github.com/maistra/api/blob/maistra-2.1/manifests/maistra.io_meshfederations.yaml#L6